### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by Mallets**
Fix docs

### CODE REVIEW

File: zenoh/src/api/queryable.rs [hunk 1]
```diff
@@ -132,7 +132,7 @@ impl Query {
         self.attachment.as_mut()
     }
 
-    /// Sends a reply in the form of [`Sample`] to this Query.
+    /// Send a reply in the form of [`Sample`] to this Query.
```
Comment (fuzzypixelz) [inline zenoh/src/api/queryable.rs:135]:
```suggestion
    /// Sends a reply in the form of [`Sample`] to this Query.
```

I would stick with the third person pronoun as is done in `std` and most Rust crates.

Here's an example: https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.push

File: zenoh/src/api/queryable.rs [hunk 2]
```diff
@@ -147,7 +147,7 @@ impl Query {
         }
     }
 
-    /// Sends a reply to this Query.
+    /// Send a put as a reply to this Query.
```
Comment (fuzzypixelz) [inline zenoh/src/api/queryable.rs:150]:
```suggestion
    /// Sends a [`crate::api::sample::Sample`] of kind [`crate::api::sample::SampleKind::Put`] as a reply to this Query.
```

File: zenoh/src/api/queryable.rs [hunk 3]
```diff
@@ -191,7 +190,7 @@ impl Query {
         }
     }
 
-    /// Sends a delete reply to this Query.
+    /// Send a delete as a reply to this Query.
```
Comment (fuzzypixelz) [inline zenoh/src/api/queryable.rs:193]:
```suggestion
    /// Sends a [`crate::sample::Sample`] of kind [`crate::sample::SampleKind::Delete`] as a reply to this Query.
```

**Review by fuzzypixelz — CHANGES_REQUESTED** [review]
I would stick with the third person pronoun as is done in `std` and most Rust crates.

Here's an example: https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.push

**Comment by Mallets** [comment]
@fuzzypixelz  Suggestions applied!

**Review by fuzzypixelz — APPROVED** [review]

**Pull request merged**